### DISCUSSION TITLE
What's  a VERIFIER while using github oauth2 ?

### DISCUSSION THREAD

**Discussion by Mr-M1M3** [pocketbase/pocketbase#3245] [Q&A]
Hello, I'm completely new to oauth2 integration. What I understood by reading the `Auth with OAuth2` section of the docs is I have to pass a parameter named `VERIFIER`. However, till now, I haven't encountered with such a term while using github oauth2 (I tried to do it without pocketbase first).

What's that `VERIFIER`?

**Comment by ganigeorgiev** [comment]
`codeVerifier` and `codeChallenge` are used with the OAuth2 [PKCE](https://oauth.net/2/pkce/) flow.

I think GitHub currently doesn't support the PKCE flow so it is ignored there.
